ocaml/xapi/xapi_globs.ml

```diff
@@ -935,7 +935,7 @@ let depmod = ref "/usr/sbin/depmod"
 
 let driver_tool = ref "/opt/xensource/debug/drivertool.sh"
 
-let dracut = ref "/usr/sbin/dracut"
+let dracut = ref "/usr/bin/dracut"
```

Comment:
This one is unused, normally I'd say delete it. But I think this was introduced recently as part of driver multi-versioning, so perhaps we plan on using it later, so lets keep it for now.